<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tezhenzhiClass</class>
 <widget class="QDialog" name="tezhenzhiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tezhenzhi</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="check1">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>重定位已经分离</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check2">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>可执行镜像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check3">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>行号已分离</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check4">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>本地符号已分离</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check5">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>主动WS修饰</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check6">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>留意大地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check7">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>字节保留为低</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check8">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>期望是32位系统</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check9">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>调试信息已分离</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check10">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>交换区运行（可移动设备）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check11">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>交换区运行（网络）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check12">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>文件系统</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check13">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>DLL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check14">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>非多处理器系统</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check15">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>字节保留为高</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>tezhenzhiClass</receiver>
   <slot>SetTzz()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>tezhenzhiClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetTzz()</slot>
 </slots>
</ui>
